CollateralAccount/CollateralAccount.md

```diff
@@ -0,0 +1,106 @@
+1. Multisig wallets don't have any non-script addresses in general and so would need some separate solution
+
+So with this, we can see the naive solution is complicated while still having issues and an unintuitive user experience.
+
+
+
+# Specification
+
+Recall that [CIP1852](../CIP-1852) specifies the following derivation path
+
+```
+m / purpose' / coin_type' / account' / chain / address_index
+```
+
+We set `chain=3` to indicate the *collateral chain*.
+
+## *address_index* value
+
+Wallets MUST only use `address_index=0` for this specification. Since a single address can contain multiple UTXO entries, there is no need to derive other addresses (using more addresses would not provide any privacy benefit and would complicate address discovery).
+
+We will call this specific derivation the *collateral account*.
+
+## New solution for managing collateral
+
+Wallets SHOULD only use UTXO entries of the collateral account as collateral in the transaction. If the collateral is insufficient, the wallet should tell the user to send more ADA to their collateral account.
+
+The benefits of this solution are as follows:
+1. Transactions in the history to add collateral can be clearly marked as such by wallet software.
+1. Prompting the user to add ADA to their collateral account doesn't require explaining users about how UTXO works under the hood compared to the refactoring option.
+1. No change required to input selection algorithms
+1. Multisig (or other script-based wallets) are handled the same way as regular wallets
+1. User only risks as much collateral as their are comfortable with
+
+However, this solution also comes with downsides:
+
+1. Reusing the same collateral account for all your smart contract calls gets rid of any privacy. However, base addresses in Cardano already forgo privacy and so users who want to call smart contracts with privacy will already need to handle things differently.
+1. The blockchain can't stop people from sending tokens to somebody's collateral account. This doesn't harm the user (since other UTXO entries can still be used) and wallets will already need to implement a way for the user to withdraw from their collateral account so the impact should be minimal.
+1. It's possible for the collateral account to contained mangled addresses (same as regular base addresses for the wallet), but empirically mangled addresses are rare and people should not be sending to collateral accounts directly anyway so this is of minimal concern
+1. Support for a collateral account needs to be added to hardware wallets
+1. If the user doesn't reduce the ADA in their collateral account when calling cheaper smart contracts, they are still be putting up more collateral than necessary
```

Comment:
I feel like there could be simple improvements to handle this, maybe some mechanism to take an average of collateral within the network to help users better 'forecast' what collateral they might need so they can plan ahead and not use more than necessary.

Reply:
Yes, wallets should also make a suggestions for how much to put in for common use cases. Something like "use DEXs often? Put this much in and you'll usually be fine!". We can warn them before they make a smart contract call to add more collateral, but it's ideal if they just set some amount upfront for their usage pattern and forget about it

Reply:
The proposal implies wallets would always select all "pure" UTxO available on the collateral account?

Reply:
Probably best not to select more than required because otherwise you're increasing the tx fee for no reason. Additionally, there's a limit on the number of UTXO entries you can add as collateral.